packages/app/components/FormLogin/index.tsx

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from "react";
+import { useEffect, useState } from "react";
+import { View } from "react-native";
+import { useTranslation } from "app/common-i18n/use-translation";
+import styled, { css } from "styled-components/native";
+
+import { ButtonCta, ButtonSM } from "../Buttons";
+import CompositionSection from "../Compositions/CompositionSection/index.web";
```

Comment:
You work with mobile application, but make import for web. If you need this component for native, create it